### PR TITLE
Update Pull Requests

### PR DESCRIPTION
**Pull request by jasonLaster**
I feel like we've been doing some really great things in our PRs which deserve to be documented.

* Added more content to screenshots
* Added a section for updates, which i've wanted for awhile

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -112,13 +112,13 @@ When a person is assigned to an issue this indicates an _intent to implement_.
...
-* Include screenshots and animated GIFs in your pull request whenever possible.
-* List any steps necessary to trigger the feature you've created or bug you are fixing
-* Always run the [unit tests](#unit-tests) locally before creating your PR
- * The [integration tests](#integration-tests) will be run automatically by the CI or you can try running them locally as well
-* Once the tests have passed in the PR you must receive a review using the GitHub review system
- * To learn more about GitHub reviews take a look at their [documentation](https://help.github.com/articles/reviewing-changes-in-pull-requests/) and [video tutorial](https://youtu.be/HW0RPaJqm4g)
-* We have a number of contributors reviewing PRs fairly quickly, if you feel yours has been neglected please mention the team name **@devtools-html/debugger** in the PR
+Go to [Pull Requests](./docs/pull-requests.md) to learn about:
+
+* [Screenshots](./docs/pull-requests.md#Screenshots)
```
Comment (clarkbw) [inline CONTRIBUTING.md]: I think you need to lowercase `#screenshots` and the other links.

File: docs/pull-requests.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+We value landing PRs smoothly. One way we minimize back and forth, is by sometimes pushing updates directly to the PR branch.
+
+There are a couple times when we do this:
+* it's a small syntax or style change that's blocking a merge
+* we want to suggest a refactor. At this point, feel free to offer your opinion.
+
+*For Reviewers*
+
+There are a couple of github requirements for pushing to a PR branch:
```
Comment (clarkbw) [inline docs/pull-requests.md]: I'm not sure I understand this section.  This is config / setup for reviewers?  people on the @debugger team?

File: docs/pull-requests.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+1. **http remote** `https://github.com/wldcordeiro/debugger.html.git`
```
Comment (clarkbw) [inline docs/pull-requests.md]: Is this supposed to be `https://github.com/devtools-html/debugger.html.git`?

File: docs/pull-requests.md (hunk 2 above)
Comment (jasonLaster) [inline docs/pull-requests.md]: Oh yeah, this is probably not clear. I'm trying to help team members like you or wellington get setup

**Pull request merged**